lib/game_box_web/helpers.ex

```diff
@@ -0,0 +1,43 @@
+defmodule GameBoxWeb.Helpers do
```

Comment:
I think this module could use a more specific name. Perhaps something like `ComponentHelpers? I think it could be helpful so that it doesn't become a catch-all for re-usable things that we don't have a good home for yet.

Reply:
Makes sense, I'll change that.